lib/TabTracker.js

```diff
@@ -117,10 +117,12 @@ TabTracker.prototype = {
   handlePerformanceEvent(eventData, eventName, value) {
     let payload = Object.assign({}, eventData);
     payload.action = "activity_stream_performance";
-    payload.tab_id = tabs.activeTab.id;
     payload.event = eventName;
     payload.value = value;
-    this._setCommonProperties(payload, tabs.activeTab.url);
+    if (tabs.activeTab) {
```

Comment:
I think we can just short circuit here if there is no activeTab property in tabs, cause it's pointless to send the telemetry in that case.